fix_telegram_webhook_config.py

```
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)